### ISSUE TITLE
ypography Disaster - Unreadable Text

### ISSUE THREAD

**Issue by Ayaanshaikh12243** [Jagrati3/Environment_Animal_Safety_Hub#1232]
#1203
Priority: Medium

Poor Font Loading: FOIT (Flash of Invisible Text) for 2-3 seconds on slow connections

Contrast Issues: Light gray text on white background fails WCAG standards

Font Size Problems: Text too small on mobile (below 16px minimum)

Line Height Issues: Cramped text with poor readability

Font Weight Overuse: Loading 6 font weights but only using 2
